program/src/pod_instruction.rs

```diff
@@ -115,6 +115,9 @@ pub(crate) enum PodTokenInstruction {
     ConfidentialMintBurnExtension,
     ScaledUiAmountExtension,
     PausableExtension,
+    // 45
+    PermissionedBurnExtension,
+    PermissionedBurn,
```

Comment:
Sorry, this wasn't clear, but can you add `PermissionedBurn` as a new instruction under `PermissionedBurnExtension`? And maybe we can just call it `Burn` or `BurnChecked`